module-i18n.php

```diff
@@ -5,6 +5,8 @@
 	_x( 'Adds support to store the dominant color of newly uploaded images and create a placeholder background of that color.', 'module description', 'performance-lab' ),
 	_x( 'WebP Support Health Check', 'module name', 'performance-lab' ),
 	_x( 'Adds a WebP support check in Site Health status.', 'module description', 'performance-lab' ),
+	_x( 'Partytown Web Worker', 'module name', 'performance-lab' ),
+	_x( 'Run JavaScript in a separate Web Worker with the help of Partytown library.', 'module description', 'performance-lab' ),
```

Comment:
```suggestion
	_x( 'Add support for running JavaScript in a separate Web Worker with the help of the Partytown library.', 'module description', 'performance-lab' ),
```

Reply:
I think it's an auto-generated file. Better to update this in the plugin header?

Reply:
Wouldn't it be in both? Curious why `module-i18n.php` is committed to git in the first place. Seems like it should be part of the build process only? cc @mukeshpanchal27